src/main/kotlin/no/nav/yrkesskade/meldingmottak/integration/mottak/model/SkadeforklaringInnsendingHendelse.kt

```diff
@@ -1,8 +1,10 @@
 package no.nav.yrkesskade.meldingmottak.integration.mottak.model
 
+import no.nav.yrkesskade.meldingmottak.domene.BeriketData
 import no.nav.yrkesskade.meldingmottak.integration.model.Skadeforklaring
 
 data class SkadeforklaringInnsendingHendelse(
```

Comment:
Bare lurte litt på hvorfor denne klassen ikke ligger i yrkesskade-backend-felles?

Reply:
Ja, snakket om det med Jone på starten av standup i dag. Nå ligger modellen både i skadeforklaring-api og i mottak, men bør flyttes til backend-felles. Grunnen til at det ble sånn var fordi Jone bruker openapi generator, og for å få til raske iterasjoner ved endringer i kontrakten.